cider-velox/src/ciderTransformer/CiderPlanPatterns.h

```diff
@@ -152,4 +160,52 @@ class PartialAggPattern : public SequencePlanPattern {
   PartialAggPattern() { setStateMachine(std::make_shared<PartialAggStateMachine>()); }
 };
 
+class OrderByStateMachine : public StateMachine {
```

Comment:
do we have refactored plan pattern's develop guide as without state machine?

Reply:
no